<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookStackedPage</class>
 <widget class="QWidget" name="BookStackedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="TypeBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>书名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>isbn</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="ViewButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索书籍</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ViewTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>651</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>图书馆</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>isbn</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ChangeButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
